tests/autorestart/test_container_autorestart.py

```diff
@@ -29,10 +29,14 @@
 
 
 @pytest.fixture(autouse=True, scope='module')
-def config_reload_after_tests(duthosts, selected_rand_one_per_hwsku_hostname):
+def config_reload_after_tests(duthosts, selected_rand_one_per_hwsku_hostname, tbinfo):
     # Enable autorestart for all features before the test begins
     for hostname in selected_rand_one_per_hwsku_hostname:
         duthost = duthosts[hostname]
+        # Enable dhcp_server feature for mx topo
+        if tbinfo["topo"]["type"] == "mx":
+            duthost.shell("config feature state dhcp_server enabled")
```

Comment:
Maybe it would be better to add `module_ignore_errors=True` in shell function to cover that dhcp_server is not included in image?